_posts/2022-02-18-Log4Shell-the-day-when-Security-industry-was-working-overtime.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: post
+author: keitsi
+og_image: /img/posts/log4shell/logo.png
+tag: Blog post
+description:
+  This blog is about the series of Log4Shell vulnerabilities, how they
+  affected our software development and how we helped our customer to
+  validate their networks. I hope this blog will give the reader an 
+  insider view of what it is actually like to patch and detect those
+  vulnerabilities.
+title:
+  Log4Shell Vulnerability - the day when Security industry was working overtime
+excerpt:
+  This blog is about the series of Log4Shell vulnerabilities, how they
+  affected our software development and how we helped our customer to
+  validate their networks. I hope this blog will give the reader an 
+  insider view of what it is actually like to patch and detect those
+  vulnerabilities.
+---
+This blog is about the series of Log4shell vulnerabilities best known as CVE-2021-44832, how they affected our software development and how we helped our customer to check their networks. I hope this blog will give the reader an insider view of what it is actually like to patch and detect those vulnerabilities.
```

Comment:
Run Prettier for the file. Looks like new versions change formatting a bit so better to run only for this file. I'll fix globally at some point.